etl/steps/data/garden/ggdc/2022-11-28/penn_world_table_national_accounts.py

```diff
@@ -0,0 +1,76 @@
+import json
+from typing import List, cast
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from owid.catalog.utils import underscore_table
+from owid.datautils import geo
+from structlog import get_logger
+
+from etl.helpers import Names
+from etl.paths import DATA_DIR
+
+log = get_logger()
+
+# naming conventions
+N = Names(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("penn_world_table_national_accounts.start")
+
+    # read dataset from meadow
+    ds_meadow = Dataset(DATA_DIR / "meadow/ggdc/2022-11-28/penn_world_table_national_accounts")
+    tb_meadow = ds_meadow["penn_world_table_national_accounts"]
+
+    df = pd.DataFrame(tb_meadow)
+
+    log.info("penn_world_table_national_accounts.exclude_countries")
+    df = exclude_countries(df)
+
+    log.info("penn_world_table_national_accounts.harmonize_countries")
+    df = harmonize_countries(df)
```

Comment:
If I understand correctly, your country names are already harmonized, and hence you can delete the last two lines.